### PR TITLE
B2B-1494: FE Decode the styles from the iframe url and pass them into the Link UI

### PR DESCRIPTION
**Pull request by vitaliimovchan-front**
https://front-finance.atlassian.net/browse/B2B-1494

### How to test
```js
// App.tsx line 38
const link = window.atob(data.content.linkToken)
setLinkToken(window.btoa(link + '&link_style=eyJpciI6IDIsICJpbyI6IDAuOH0='))
```

### CODE REVIEW

File: packages/link/src/FrontConnection.test.ts [hunk 1]
```diff
@@ -48,20 +48,6 @@ describe('createFrontConnection tests', () => {
     )
   })
 
-  test('createFrontConnection when valid link provided should open popup', () => {
```
Comment (vitaliimovchan-front) [inline packages/link/src/FrontConnection.test.ts:51]: this was a duplicate

File: packages/link/src/utils/popup.test.ts [hunk 2]
```diff
@@ -2,19 +2,19 @@ import { addPopup, removePopup } from './popup'
 
 describe('Popup tests', () => {
   test('addPopup should add correct popup', () => {
-    addPopup('http://localhost/1')
+    const link = 'https://some.domain&link_style=eyJpciI6IDIsICJpbyI6IDAuOH0='
```
Comment (salavatgainetdinov-front) [inline packages/link/src/utils/popup.test.ts:5]:
```suggestion
    const link = 'https://some.domain?link_style=eyJpciI6IDIsICJpbyI6IDAuOH0='
```

File: packages/link/src/utils/style.test.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { getLinkStyle, getNumber } from './style'
+
+describe('Test style utils', () => {
+  test('verify function returns a style object', () => {
+    const link = 'https://some.domain&link_style=eyJpciI6IDI0LCAiaW8iOiAwLjF9'
```
Comment (salavatgainetdinov-front) [inline packages/link/src/utils/style.test.ts:5]:
```suggestion
    const link = 'https://some.domain?link_style=eyJpciI6IDI0LCAiaW8iOiAwLjF9'
```

File: packages/link/src/utils/style.test.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    const received = getLinkStyle(link)
+    expect(received).toEqual({ ir: 24, io: 0.1 })
+  })
+
+  test('verify function returns nothing on missing query param', () => {
+    const link = 'https://some.domain&other=12'
```
Comment (salavatgainetdinov-front) [inline packages/link/src/utils/style.test.ts:11]:
```suggestion
    const link = 'https://some.domain?other=12'
```

File: packages/link/src/utils/style.test.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    const received = getLinkStyle(link)
+    expect(received).toBeNull()
+  })
+
+  test('verify function returns nothing on wrong encoded value', () => {
+    const link = 'https://some.domain&link_style=eyJpciI6IDI0LCAiaW8iOiAwL'
```
Comment (salavatgainetdinov-front) [inline packages/link/src/utils/style.test.ts:17]:
```suggestion
    const link = 'https://some.domain?link_style=eyJpciI6IDI0LCAiaW8iOiAwL'
```

File: packages/link/src/utils/popup.ts [hunk 6]
```diff
@@ -13,7 +16,7 @@ const getPopupHtml = (link: string) => `
 </div>
 `
 
-const styles = `
+const styles = (style?: LinkStyle) => `
```
Comment (salavatgainetdinov-front) [inline packages/link/src/utils/popup.ts:19]:
```suggestion
const getStyles = (style?: LinkStyle) => `
```

**Review by salavatgainetdinov-front — APPROVED** [review]

**Pull request merged**